webapp/src/components/training/TrainerDashboard.vue

```diff
@@ -1,15 +1,15 @@
 <template>
   <div class="space-y-4 md:space-y-8">
     <!-- Train Button -->
-    <div class="w-full lg:w-1/2 lg:max-w-[640px] mx-auto">
+    <div class="w-full lg:w-1/2 lg:max-w-[640px] mx-auto tuto-train-dash">
       <IconCard title-placement="center">
         <template #title> Control the Training Flow </template>
           <!-- If we are not currently training -->
           <div v-if="!isTraining" class="flex flex-col gap-y-4">
             <!-- Toggle buttons between training collaboratively and locally -->
             <div class="flex justify-center">
               <button
-                id="train-collaboratively-bttn"
+                id="tuto-train-collab"
```

Comment:
theses IDs (`train-collaboratively-bttn` and `training-locally-bttn`) are still used in TrainingSteps.vue

in here, I don't think renaming to "tuto" prefixed id makes sense as the IDs are not solely used w/ the tuto (used in TrainerDashboard.spec.ts)

Reply:
I don't see where `train-collaboratively-bttn` and `training-locally-bttn` are used in TrainingSteps.vue?
I've renamed them to `train-[local,collab]-bttn`.